docs/example-react-router.mdx

```diff
@@ -3,7 +3,159 @@ id: example-react-router
 title: React Router
 ---
 
-This example demonstrates React Router v6. For previous versions see below.
+This example demonstrates React Router v6.4 and above. For previous versions see below.
```

Comment:
I'm not very familiar with this, but I assume `createBrowserRouter` is added in v6.4?